## RUB1 (Rome Unseminar in Bioinformatics)

Please enter sections here and then items for individual tools (Please provide a link if possible):
    
### Mass Spec

- Machine related data processing
- differs from machine to machine
- ex Bruker: Flex analysis Biotyper, Waters: Progenesis, Thermo: Proteome discoverer

### Database search

- ex MASCOT http://www.matrixscience.com/search_form_select.html
- XTandem https://www.thegpm.org/tandem/, 
- SEQUEST http://proteomicswiki.com/wiki/index.php/SEQUEST, 
- Andromeda, SearchGUI/PeptideShaker
- BYONIC https://www.proteinmetrics.com/products/byonic/, 
- PEAKSDb http://www.bioinfor.com/peaksdb/

### Quantiffication

- SCAFFOLD http://www.proteomesoftware.com/products/scaffold/, 
- MaxQuant https://www.biochem.mpg.de/5111795/maxquant, 
- SKYLINE (targeted proteomics) https://skyline.ms/wiki/home/software/Skyline/page.view?name=default

### Pathways

- KEGG pathway database linked with BLAST2GO,
- IPA https://www.qiagenbioinformatics.com/products/ingenuity-pathway-analysis/

### Cross-linking Mass Spec

- ex MeroX https://www.stavrox.com/
- XLinkX https://www.hecklab.com/software/xlinkx/,
- xiNET/xiVIEW http://crosslinkviewer.org/
- Proteome Discoverer/Progenesis
- MaxQuant


### RNA/protein structure prediction

- Tools that predict tertiary structure given the primary and secondary structure
- RNAComposer (http://rnacomposer.cs.put.poznan.pl/)

### Tools that compare different 3D prediction with the original ones

- RNAsses (http://rnassess.cs.put.poznan.pl/)
- Tools that predict secondary structure from single primary sequence
- Dotknot (http://dotknot.csse.uwa.edu.au/)
- Pkiss (https://bibiserv2.cebitec.uni-bielefeld.de/pkiss)
- RNAFold (http://rna.tbi.univie.ac.at/cgi-bin/RNAWebSuite/RNAfold.cgi)
- mfold (http://unafold.rna.albany.edu/?q=mfold)
- RNAStructure (https://rna.urmc.rochester.edu/RNAstructureWeb/)
- IPknot (https://rtips.dna.bio.keio.ac.jp/ipknot/)
- CONTRAFold (http://contra.stanford.edu/contrafold/server.html)
- CentroidFold (http://rtools.cbrc.jp/centroidfold/)

### Tools that extract secondary structure information from PDB data:

- RNApdbee (http://rnapdbee.cs.put.poznan.pl/)

### Tools that visualize RNA secondary structure

- VARNA (http://varna.lri.fr/)
- forna (http://rna.tbi.univie.ac.at/forna/)
- Protein prediction: (insert primary sequence)
- Phyre2


### Network analysis

- gene ontology  (http://www.geneontology.org/)
- string https://string-db.org/
- blast2go https://www.blast2go.com/
- cytoscape https://cytoscape.org/
- clueGO http://apps.cytoscape.org/apps/cluego
- phosphopath http://apps.cytoscape.org/apps/phosphopath
- biogrid https://thebiogrid.org/
- arena3d (http://arena3d.org/)
- intact  https://www.ebi.ac.uk/intact/
- clusterMaker (app) http://www.cgl.ucsf.edu/cytoscape/cluster/clusterMaker.shtml
- (3b) binGO(app) https://www.psb.ugent.be/cbd/papers/BiNGO/Home.html
- MCL clustering algorithm https://micans.org/mcl/
- intact score
- corum (https://mips.helmholtz-muenchen.de/corum/)
- webgestalt (http://www.webgestalt.org/option.php)
- cytoscapejs
- stringapp http://apps.cytoscape.org/apps/stringapp
- networkX

### Docking 

Docking is a computational technique that allows to predict the potential contact zone/binding mode of a macromolecule with a ligand. The ligand can be a protein, peptide, small molecule, DNA, RNA,... It predicts the conformation with the minimal energy at the interface between the macromolecule and ligand.

- Haddock, Autodock, Autodock vina

- protein ligand docking 
  - autodock free (http://autodock.scripps.edu/) 
  - autodock vina: protein-ligand and virtual screening (http://vina.scripps.edu/manual.html)
  - idock (http://istar.cse.cuhk.edu.hk/idock/)
  - flexaid (http://biophys.umontreal.ca/nrg/NRG/FlexAID.html)
  - glide (https://www.schrodinger.com/glide)

- protein protein docking 
  - haddock
  - zdock
  - gramm-x  
  - gold 
  - bioluminate (protein-protein / protein-peptide)
  - database of compounds 
  - ambinter 
  - zinc15
  - webserver 










